lib/src/spawner.rs

```diff
@@ -110,14 +114,12 @@ impl Spawner {
             builder.insert(ConstantVelocity::new(facing.to_vec3() * speed));
         }
 
-        // Housekeeping
-        let new_hitbox = builder.id();
         if descriptor.attached_to_player {
             commands.entity(parent).push_children(&[new_hitbox]);
         }
-        self.spawned.insert(id, new_hitbox);
+        self.spawned.insert(new_hitbox.id(), new_hitbox);
```

Comment:
new_hitbox pitäis olla jo id, koska rivi 99. effectively kutsut builder.id().id()

Kääntyykö tää?

Reply:
Kääntyy ainakin minulla

Reply:
Kyl kääntyy mullakin.

Näemmä builder.id() antaa sen id:n (tyyppiä Entity) jonka uus entity on saamassa, mutta Entityllä on .id() jota voi myös käyttää id:nä. IMO lukijalle Entity on parempi koska se on explicitly selkeä id Entitylle, missä u32 voi olla anything.